index.html

```diff
@@ -237,7 +237,7 @@ <h3>
             <dd>
               <p>
 This family of specifications consists of documents that each define how to
-express proofs of integrity for verifiable credentials using a number of
+express and associate proofs of integrity with data, focusing on verifiable credentials and
```

Comment:
Alternate # 1: 

```suggestion
express and associate proofs of integrity with well-bounded datasets, focusing on verifiable credentials and
```

Alternate # 2:

```suggestion
express and associate proofs of integrity with digital documents, focusing on verifiable credentials and
```

Reply:
I would be ok with "focusing only on verifiable credentials".

Reply:
> I would probably be ok with "focusing only on verifiable credentials".

Ok, I think that compromise might be fine. To be clear, you'd be ok w/ this language?

> express and associate proofs of integrity with well-bounded datasets, focusing only on verifiable credentials

(modulo an alternative to "well-bounded datasets" that everyone can agree to, like "document-bounded datasets -- like @dlongley suggested during the call today).

Reply:
^ I would not be ok with this, unless we drop the data integrity work as a whole. We either do it well, or we don't do it... I am against "letting it in" and then "crippling it" by forcing to be only about VCs.

Reply:
> We either do it well, or we don't do it... I am against "letting it in" and then "crippling it" by forcing it to be only about VCs.

I agree with your desire, @OR13 -- that's exactly what I'm concerned about.

That said, what part of @Sakurann's suggestion makes you feel like by adding the word "only", we run the risk of crippling the work? I guess one interpretation of "only" is that we're "forbidden to talk about Data Integrity in any other context other than VCs?"... so if someone was like: "Oh, if we do that in the Data Integrity work, that'll make it impossible to do X in YZWG!", then someone could respond with "Our charter doesn't allow us to contemplate how Data Integrity applies to YZWG!".

Reply:
> I guess one interpretation of "only" is that we're "forbidden to talk about Data Integrity in any other context other than VCs?"

Thats exactly how I read the desire to add the word only... in other words, its crippling.

Reply:
Adding the term "well-bounded datasets" would make things worse. What do we mean by "well-bounded datasets"? How would we decide/define whether dataset XYZ coming from ZZWG is well-bounded? What happens if it is: do we have then a charter obligation to fulfil the requirements of the ZZWG even if the dataset XYZ has absolutely nothing to do with VC data? A definite -1 from me adding that term.

Reply:
> Adding the term "well-bounded datasets" would make things worse.

What about "documents" or "documents (well-bounded digital objects containing information)"?

Reply:
Define well-bounded? In any relevant context?

Reply:
Well, what about leaving the term "well bounded" out and just saying "documents" or "digital documents"? It avoid the thing DanB was concerned about wrt. "arbitrarily large graphs", right?

Reply:
The problem is: however we try, it is a non-trivial task to define the class of datasets for which our technologies should be applicable. We all have a vague "feeling" of what they are, but coming up with a precise terminology out of the blue for the purpose of this charter will not work. May I remind you that, in the community we are talking about, we also have very mathematical minds who would criticize such an attempt in excruciating details (and for good reasons)? Do we need that here and now?  I do not think so.